Misc/NEWS.d/next/Core and Builtins/2023-12-19-17-27-08.gh-issue-111375.XeHOl_.rst

```diff
@@ -0,0 +1,2 @@
+Add sys._set_exception() function that can set/clear the current exception
+context. Patch by Carey Metcalfe.
```

Comment:
We have 3 news files here. This and the next one should be deleted.

Reply:
Will do. Should the `sys._exception()` be added to the other news file or just not noted at all since it's an "internal" function? (provided `sys_exception()` is how this ends up being implemented, otherwise this is a moot point)

Reply:
Probably not needed if it's private. But you can wait with the news file till we know what's going in.